app/src/main/java/org/mozilla/focus/shortcut/IconGenerator.kt

```diff
@@ -23,6 +25,7 @@ class IconGenerator {
     companion object {
         private val TEXT_SIZE_DP = 36f
         private val DEFAULT_ICON_CHAR = '?'
+        private val SEARCH_ICON_FRAME = 0.15
```

Comment:
\[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.